docs/intro/building.rst

```diff
@@ -63,7 +63,8 @@ The code can be built as
 .. code-block:: bash
 
     mkdir build && cd build
-    cmake -DCMAKE_PREFIX_PATH="<install_dir>/kokkos;<install_dir>/kokkos_fft" ..
+    cmake -DCMAKE_CXX_COMPILER=icpx \
```

Comment:
Why advertise for the Intel compiler?

You could be more generic with `-DCMAKE_CXX_COMPILER=<your C++ compiler>`.

Reply:
Then `g++`? I would like to make it as easy as possible, i.e., ideally, copy and paste this part is sufficient.

Reply:
Then don't put anything. CMake would check the environment variable `CC`, and ultimately default to GCC.

Reply:
I fixed accordingly. I will merge this and make a release candidate. Thank you for your review